code/modules/modular_computers/computers/item/tablet.dm

```diff
@@ -137,6 +137,24 @@
 	comp_light_luminosity = 6.3
 	has_variants = FALSE
 
+/obj/item/modular_computer/tablet/syndicate_contract_uplink/preset/uplink/Initialize(mapload)
```

Comment:
`/obj/item/modular_computer/tablet/syndicate_contract_uplink/preset/uplink` is the only version of the contractor tablet that's actually used anywhere, so there's no reason it can't all just be moved onto the base type.
```suggestion
/obj/item/modular_computer/tablet/syndicate_contract_uplink/Initialize(mapload)
```
(Plus any other references to `/preset/uplink` of course.)